decidim-system/app/cells/decidim/system/system_checks_cell.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Decidim
+  module System
+    class SystemChecksCell < Decidim::ViewModel
+      def show
+        render
+      end
+
+      private
+
+      def checks
+        {
+          secret_key: {
+            check_method: correct_secret_key_base?,
+            error_extra: generated_secret_key
+          },
+          active_job_queue: {
+            check_method: correct_active_job_queue?,
+            error_extra: active_job_queue_link
+          }
+        }
+      end
+
+      def correct_secret_key_base?
+        Rails.application.secrets.secret_key_base.length == 128
```

Comment:
IN order to avoid an `undefined method 'length' for nil:NilClass` error, you will need to add safe navigation.

```suggestion
        Rails.application.secrets.secret_key_base&.length == 128
```

Reply:
Fixed in bdc9e73f29bb22a0a096e4c8876b104e79eab34a 

(In the timeline is before because I had forgotten to git push 🙈)